### DISCUSSION TITLE
Provision problem

### DISCUSSION THREAD

**Discussion by n-ramos** [deployphp/deployer#3361] [Help needed] [answered]
- Deployer version: 7.0.2
- HOST OS: Ubuntu 20.04.5 LTS (GNU/Linux 5.4.0-126-generic x86_64)
- Deployment OS: MAC OS monterey 12.3.1

      Hello, on launch command vendor/bin/dep provision -o remote_user=root

i have got an error: 

```
 Reading package lists...
 run curl -1sLf 'https://dl.cloudsmith.io/public/caddy/stable/gpg.key' | gpg --dearmor -o /usr/share/keyrings/caddy-stable-archive-keyring.gpg
[ gpg: cannot open '/dev/tty': No such device or address
(23) Faile
d writing body
 error  in provision.php on line 87:
exit code 2 (Misuse of shell builtins)

```

Did i do a mistake ?

Bye

**Comment by thomasnavarro** [comment, marked as answer]
Hi, I just submitted a [pull request](https://github.com/deployphp/deployer/pull/3407) about this.

Thomas

**Reply by n-ramos** [reply]
Great thanks
